new/analyse/create-package-json-content/get-package-json.ts

```diff
@@ -0,0 +1,64 @@
+import { Env, PackageAfterCreatePackageJsonContent } from '../../types';
+
+import { clone } from 'ramda';
+
+function getPackageJson(
+  packages: Env['packages'],
+  packageName: string,
+  type: 'nexus'
+): PackageAfterCreatePackageJsonContent['packageJsonNexus'];
+function getPackageJson(
+  packages: Env['packages'],
+  packageName: string,
+  type: 'git'
+): PackageAfterCreatePackageJsonContent['packageJsonGit'];
```

Comment:
Both types are supposed to be identical.

Reply:
in the end, they're both `PackageJson` from `import { CoreProperties as PackageJson } from '@schemastore/package';`

Reply:
Fine with that :-)